schema.js

```diff
@@ -26,9 +26,14 @@ module.exports = Joi.object().keys({
     localization: Joi.object().keys({
       flipNumberAndStreetCountries: Joi.array().items(Joi.string().regex(/^[A-Z]{3}$/))
     }).unknown(false),
-    pipService: Joi.string().uri({ scheme: /https?/ }),
+    pipService: Joi.any().forbidden(), // got moved to services
```

Comment:
If you could, keep this change out for a little bit. That way we can leave the old `pipService` definition in the production config, but add the new one, and then transition over more smoothly. I think we could handle updating the API code and config at the same time, but its easier if we don't have to worry about it.